### PR TITLE
Add some additional details on how to edit plugin code

### PR DESCRIPTION
**Pull request by mit-mit**
Adding a from more details to what started in https://github.com/flutter/website/pull/561

### CODE REVIEW

File: platform-plugins.md [hunk 1]
```diff
@@ -158,36 +158,56 @@ This creates a plugin project in the `hello/` folder with the following content:
...
+#### Plugin API code (.dart)
+
+To edit the Dart plugin API code, open `hello/` in IntelliJ IDEA (or your
+favorite Dart editor), The plugin API is located in `lib/main.dart` shown in the 
+Project view.
+
+To run the plugin, you need to launch the plugin example app, which requires
+defining a launch configuration:
+
+1. Select 'Run > Edit Configurations...'.
```
Comment (goderbauer) [inline platform-plugins.md]: Could the configuration be generated by `flutter create --plugin`?

File: platform-plugins.md [hunk 2]
```diff
@@ -158,36 +158,56 @@ This creates a plugin project in the `hello/` folder with the following content:
...
-pressing the &#9654; button.
-
-For the Dart code, you can directly open `hello/` in IntelliJ IDEA to edit the plugin code, or
-`hello/example/` to edit the example app code. In both cases, the relevant Dart code is in the `lib`
-folder in the Project view. Once again, the example app can be run from within IDEA in the
-usual way.
+#### Plugin API code (.dart)
+
+To edit the Dart plugin API code, open `hello/` in IntelliJ IDEA (or your
+favorite Dart editor), The plugin API is located in `lib/main.dart` shown in the
```
Comment (mravn-google) [inline platform-plugins.md]: ", the" or ". The" after the closing parenthesis.

File: platform-plugins.md [hunk 3]
```diff
@@ -158,36 +158,56 @@ This creates a plugin project in the `hello/` folder with the following content:
...
+Project view.
+
+To run the plugin, you need to launch the plugin example app, which requires
+defining a launch configuration:
+
+1. Select 'Run > Edit Configurations...'.
+1. Select '+' and then 'Flutter'.
+1. In 'Dart entrypoint', enter `<plugin folder>/example/lib/main.dart`.
+1. Select 'OK'.
+1. Launch the example app with 'Run' or 'Debug'.
```
Comment (goderbauer) [inline platform-plugins.md]: Will users be surprised that they can't hot reload? Maybe add a note?

File: platform-plugins.md [hunk 4]
```diff
@@ -158,36 +158,56 @@ This creates a plugin project in the `hello/` folder with the following content:
...
+
+#### Android platform code (.java)
```
Comment (goderbauer) [inline platform-plugins.md]: (not introduced by this PR) This is kinda conflicting with what is said here: https://flutter.io/intellij-ide/#edit-android-code

File: platform-plugins.md (hunk 1 above)
Comment (mit-mit) [inline platform-plugins.md]: I think so, but not sure. Tracking in https://github.com/flutter/flutter/issues/10115

File: platform-plugins.md (hunk 3 above)
Comment (mit-mit) [inline platform-plugins.md]: Added a note

File: platform-plugins.md (hunk 2 above)
Comment (mit-mit) [inline platform-plugins.md]: Fixed

Comment (mit-mit) [inline platform-plugins.md]: Fixed

File: platform-plugins.md (hunk 4 above)
Comment (mit-mit) [inline platform-plugins.md]: Yes, we need to iron that out, but prefer to wait until after I/O

File: platform-plugins.md [hunk 5]
```diff
@@ -158,36 +158,58 @@ This creates a plugin project in the `hello/` folder with the following content:
...
+Before editing the iOS platform code in Xcode, first make sure that
+the code has been built at least once (i.e., run the example app from IntelliJ, 
+or in a terminal execute `cd hello/example; flutter build ios`).
+
+Next,
+
+1. Launch Xcode
+1. Select 'File > Open', and select the `hello/example/ios/Runner.xcworkspace` file.
+
+The iOS platform code of your plugin is located in `Pods/Development Pods/hello/Classes/` in the Project Navigator.
```
Comment (collinjackson) [inline platform-plugins.md]: @goderbauer had talked with me about maybe removing the "Classes" folder. But if this is how it works currently, it's fine.

Comment (mit-mit) [inline platform-plugins.md]: Yup, I confirmed on a clean machine that this is how it's currently working!

**Pull request merged**